### ISSUE TITLE
[BUG] Console error on startup

### ISSUE THREAD

**Issue by dmrickey** [p4535992/conditional-visibility#7]
**Module Version:** v0.3.2
![image](https://user-images.githubusercontent.com/3664822/149563134-7df42be4-59db-413f-8a55-52b839d1bdff.png)
![image](https://user-images.githubusercontent.com/3664822/149563218-ffb9e596-2c7b-47ae-afed-34429fed7450.png)

**Describe the bug**
Error in console on startup
```
ConditionalVisibility.js:158 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'restrictVisibility')
[Detected 1 package: conditional-visibility]
    at new ConditionalVisibility (ConditionalVisibility.js:158:55)
    at Function.initialize (ConditionalVisibility.js:306:42)
    at readyHooks (Hooks.js:23:27)
    at conditional-visibility.js:67:5
    at Function._call (foundry.js:294:14)
    at Function.callAll (foundry.js:253:12)
    at Game.setupGame (foundry.js:5547:11)
    at async Game._initializeGameView (foundry.js:6479:5)
    at async Game.initialize (foundry.js:5415:5)
 ```

**To Reproduce**
1. Start a world in pf1
2. observe error in console
- I have a scene configured and active as well

**Expected behavior**
I don't know if this error is benign or important or not. I haven't configured anything yet and haven't yet tried. 

**Screenshots**

**Browser:** 
chrome 97.0.4692.71

**Foundry Version:**
9 242 

**Game System:** 
pf1 0.80.4

**Comment by dmrickey** [comment]
Now I'm getting this one
```
ConditionalVisibility.js:179 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'testVisibility')
[Detected 1 package: conditional-visibility]
    at new ConditionalVisibility (ConditionalVisibility.js:179)
    at Function.initialize (ConditionalVisibility.js:311)
    at readyHooks (Hooks.js:24)
    at conditional-visibility.js:68
    at Function._call (foundry.js:294)
    at Function.callAll (foundry.js:253)
    at Game.setupGame (foundry.js:5547)
    at async Game._initializeGameView (foundry.js:6479)
    at async Game.initialize (foundry.js:5415)
```

**Comment by p4535992** [comment]
This issue is closed, because the new series 0.4.X is just to different, checkout the last version of the module.
